kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/convert/BigQueryRecordConverter.java

```diff
@@ -97,7 +97,7 @@ private Object convertObject(Object kafkaConnectObject, Schema kafkaConnectSchem
         byte[] bytes = byteBuffer.array();
         return Base64.getEncoder().encodeToString(bytes);
       case ARRAY:
```

Comment:
lets rearrange these so the order is Array, Map, Struct, Bytes, Boolean ... so the organization is a little better.